honeycomb-render/src/camera.rs

```diff
@@ -0,0 +1,180 @@
+//! Module short description
+//!
+//! Should you interact with this module directly?
+//!
+//! Content description if needed
+
+// ------ IMPORTS
+
+use winit::{
+    event::{ElementState, KeyEvent, WindowEvent},
+    keyboard::{Key, NamedKey},
+};
+
+// ------ CONTENT
+
+#[rustfmt::skip]
+pub const OPENGL_TO_WGPU_MATRIX: cgmath::Matrix4<f32> = cgmath::Matrix4::new(
+    1.0, 0.0, 0.0, 0.0,
+    0.0, 1.0, 0.0, 0.0,
+    0.0, 0.0, 0.5, 0.5,
+    0.0, 0.0, 0.0, 1.0,
+);
+
+cfg_if::cfg_if! {
+    if #[cfg(target_os = "linux")] {
+        pub const SPEED_FACTOR: f32 = 0.005;
+    } else {
+        pub const SPEED_FACTOR: f32 = 0.05;
```

Comment:
Just being curious: why ?

Reply:
I tested the visualizer on my personal laptop (running MacOS) and there is a severe disparity in the camera speed at runtime (for the same value in the code). 

Aside from the OS difference, the keyboard sensitivity may also play a part, but I don't think it can explain that much of a difference.